core/roboconf-dm-templating/src/main/java/net/roboconf/dm/templating/internal/templates/TemplateWatcher.java

```diff
@@ -0,0 +1,378 @@
+/**
```

Comment:
Maybe an issue with EGit (Eclipse's Git implementation).
I just moved the files in my IDE. I did not look at how it appeared in Git commits.